x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/components/actions_log_table.tsx

```diff
@@ -110,6 +110,7 @@ const getResponseActionListTableColumns = ({
           return (
             <EuiToolTip content={UX_MESSAGES.triggeredByRule} anchorClassName="eui-textTruncate">
               <SecuritySolutionLinkAnchor
+                title={undefined}
```

Comment:
Hey, how does this work? isn't title undefined by default?

Reply:
43e08997a6a85eb7476cac763ad2c524a05ff80b